Dockerfile

```diff
@@ -9,12 +9,12 @@ COPY run.sh run.sh
 
 RUN composer --no-interaction install
 
-FROM tenforce/virtuoso:1.3.1-virtuoso7.2.2
+FROM openlink/virtuoso-opensource-7:latest
 
 ENV IMPORT_DIR /tmp/import
 ENV DBA_PASSWORD dba
 
-RUN apt-get update && apt-get install -qq -y wget curl php7.0 php7.0-zip netcat
+RUN apt-get update && apt-get install -qq -y wget curl php7.4-cli php7.4-zip netcat
```

Comment:
Investigate this in order to update to 8.1+

Reply:
As `openlink/virtuoso-opensource-7:latest` image is based on ubuntu 20.04 by default it shipped with php7.4. For installing 8.1+ php version we have to add PPA repository in our Dockerfile.